File: exercicios/codigosEmOutrasLinguagens/EX5FU/EX5FUv1.c
#include <stdio.h>

int main() {
    int num, indice, i;
    int binario[32];

    // Primeiro número
    printf("Digite o 1º número inteiro positivo: ");
    scanf("%d", &num);

    indice = 0;
    if (num == 0) {
        printf("Binário: 0\n");
    } else {
        while (num > 0) {
            binario[indice] = num % 2;
            num = num / 2;
            indice++;
        }
        
        printf("Binário: ");
        
        for (i = indice - 1; i >= 0; i--) {
            printf("%d", binario[i]);
        }
        
        printf("\n");
    }

    // Segundo número
    printf("Digite o 2º número inteiro positivo: ");
    scanf("%d", &num);

    indice = 0;
    if (num == 0) {
        printf("Binário: 0\n");
    } else {
        while (num > 0) {
            binario[indice] = num % 2;
            num = num / 2;
            indice++;
        }
        
        printf("Binário: ");
        
        for (i = indice - 1; i >= 0; i--) {
            printf("%d", binario[i]);
        }
        
        printf("\n");
    }

    // Terceiro número
    printf("Digite o 3º número inteiro positivo: ");
    scanf("%d", &num);

    indice = 0;
    if (num == 0) {
        printf("Binário: 0\n");
    } else {
        while (num > 0) {
            binario[indice] = num % 2;
            num = num / 2;
            indice++;
        }
        
        printf("Binário: ");
        
        for (i = indice - 1; i >= 0; i--) {
            printf("%d", binario[i]);
        }
        
        printf("\n");
    }

    return 0;
}

File: exercicios/codigosEmOutrasLinguagens/EX5FU/EX5FUv1.py
# Primeiro número
num = int(input("Digite o 1º número inteiro positivo: "))
binario = []

if num == 0:
    print("Binário: 0")
else:
    while num > 0:
        binario.append(num % 2)
        num = num // 2
        
    print("Binário:", end=" ")
    
    for i in range(len(binario) - 1, -1, -1):
        print(binario[i], end="")
        
    print()

# Segundo número
num = int(input("Digite o 2º número inteiro positivo: "))
binario = []

if num == 0:
    print("Binário: 0")
else:
    while num > 0:
        binario.append(num % 2)
        num = num // 2
        
    print("Binário:", end=" ")
    
    for i in range(len(binario) - 1, -1, -1):
        print(binario[i], end="")
    
    print()

# Terceiro número
num = int(input("Digite o 3º número inteiro positivo: "))
binario = []

if num == 0:
    print("Binário: 0")
else:
    while num > 0:
        binario.append(num % 2)
        num = num // 2
    
    print("Binário:", end=" ")
    
    for i in range(len(binario) - 1, -1, -1):
        print(binario[i], end="")
    
    print()